### ISSUE TITLE
Support `vw` and `vh` units in subtitles

### ISSUE THREAD

**Issue by nyuszika7h** [pbs/pycaption#234]
I have a TTML/DFXP subtitle file that pycaption can't parse due to `vw` and `vh` units:

```
CaptionReadSyntaxError: CaptionReadSy
ntaxError((ValueError('The specified value is not valid because its unit is not recognized: 80vw. The only supported units are: PIXEL, EM, PERCENT, CELL, PT'),))
```

It would be nice to have support for those units. I have attached an example file.

[example.zip](https://github.com/pbs/pycaption/files/6631271/example.zip)

**Comment by pacifier1er** [comment]
I don't believe that vw and vh are in the original TTML spec.  I do think a similar concept was adopted in TTML2, but using the unit names rw, rh as they are relative to the root container region and not the viewport.  You can see the discussion here, https://github.com/w3c/ttml2/issues/181

All this said, given our resource availability and workload, unfortunately we will not be able to tackle this request at this moment.  We hope to come back to this at a later time if and when resources permit us to.

Thanks.

**Comment by OlteanuRares** [comment]
Fixed in 2.3.6. `vw` and `vh` units are now parsed without crashing. We're aware these aren't part of the TTML spec (TTML2 uses `rw`/`rh` instead), but real-world files from third-party tools do contain them. pycaption now treats them as percentage-equivalent on read and converts to `%` on output, so they won't propagate as invalid units downstream.
